tests/bdd/features/ha/node-agent/test_path_replacement.py

```diff
@@ -158,47 +156,7 @@ def degrade_first_path():
     Docker.kill_container(TARGET_NODE_1)
     # Sleep some time to allow HA node agent detect failed path.
     # Add one extra second to make sure detection 100% happens.
-    sleep(PATH_DETECTION_TIME + 1)
-
-
-@pytest.fixture
-def publish_to_node_2(background):
-    volume = background
-    device_uri = volume.state["target"]["deviceUri"]
-
-    try:
-        ApiClient.volumes_api().del_volume_target(VOLUME_UUID)
-    except ApiException as e:
-        # Timeout or node not online
-        assert (
-            e.status == http.HTTPStatus.REQUEST_TIMEOUT
-            or e.status == http.HTTPStatus.PRECONDITION_FAILED
-        )
-
-    ApiClient.volumes_api().del_volume_target(VOLUME_UUID, force="true")
-    volume_updated = ApiClient.volumes_api().put_volume_target(
-        VOLUME_UUID, Protocol("nvmf"), node=TARGET_NODE_2
-    )
-    device_uri_2 = volume_updated.state["target"]["deviceUri"]
-    assert device_uri != device_uri_2
-    return device_uri_2
-
-
-@pytest.fixture
-def replace_failed_path_with_node2(
-    connect_to_first_path, publish_to_node_2, ha_node_instance
-):
-    ha_node_instance.api.ReplacePath(
-        pb.ReplacePathRequest(target_nqn=NEXUS_NQN, new_path=publish_to_node_2)
-    )
-
-    device = connect_to_first_path
-    desc = nvme_list_subsystems(device)
-    subsystem = desc["Subsystems"][0]
-    assert len(subsystem["Paths"]) == 1, "Second nexus must be added to I/O path"
-    assert (
-        subsystem["Paths"][0]["State"] == "live"
-    ), "Healthy I/O path has incorrect state"
+    sleep(PATH_DETECTION_TIME * 2 + 1)
```

Comment:
Why is it * 2 now? We need to be careful here as we want to make sure what timing we have.